pom.xml

```diff
@@ -62,6 +62,12 @@
       <version>1.6</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+      <artifactId>icon-shim</artifactId>
+      <version>2.0.3</version>
+      <scope>test</scope>
```

Comment:
Not sure if this is the right fix. See https://github.com/jenkinsci/plugin-pom/pull/72.